app/components/Home.js

```diff
@@ -98,21 +118,58 @@ export default class AuctionList extends Component {
     };
   }
 
+  statusText(status) {
+    if (status.equals(0)) return "Live";
+    else if (status.equals(1)) return "Cancelled";
+    else return "Completed";
+  }
+
+  statusColor(status) {
+    if (status.equals(0)) return colors.green;
+    else if (status.equals(1)) return colors.yellow;
+    else return colors.blue;
+  }
+
   render() {
+    const auctionOfInterest = this.auctions[0];
+    if (!auctionOfInterest) return null;
+
     return (
       <Wrapper>
-        <h1>Auctions ({this.auctionsLength.toString()} total)</h1>
-        <ul>
-          {this.auctions.map(auction => (
-            <li key={auction.id.toString()}>
-              <Link to={`/auction/${auction.id.toString()}`}>
-                <b>Auction {auction.id.toString()}</b> selling NFT{" "}
-                {auction.tokenId.toString()} from contract {auction.nftAddress}{" "}
-                with current highest bid: {auction.highestBid.toString()}
-              </Link>
-            </li>
-          ))}
-        </ul>
+        <Spacer size={3} />
+        <Container style={{ color: `white` }}>
+          <LeftContainer width={40}>
+            <Status>
+              {this.statusText(auctionOfInterest.status)}{" "}
+              <StatusPulse
+                active={this.statusText(auctionOfInterest.status) == "Live"}
+                color={this.statusColor(auctionOfInterest.status)}
+              />
+            </Status>
+            <Spacer size={0.5} />
+
+            <Heading>Auction #{auctionOfInterest.id.toString()}</Heading>
+            <SellerInformation>by {auctionOfInterest.seller}</SellerInformation>
+            <Spacer size={0.5} />
+
+            <Description>
+              NFT: {auctionOfInterest.tokenId.toString()}@{
+                auctionOfInterest.nftAddress
+              }
+            </Description>
+            <Divider padded={1.5} />
+            <CountDown endDate={new Date().getTime() + 100000000} />
```

Comment:
it'd be cool if countdown could be given an auction and then render itself

Reply:
Yup, this is also gonna change once we do the `endDate` calculation based on the auction itself.